packages/network/lib/uri.ts

```diff
@@ -87,7 +87,9 @@ export function addDefaultPort (urlToCheck: any) {
 }
 
 export function getPath (urlToCheck: string) {
-  return url.parse(urlToCheck).path
+  const url = new URL(urlToCheck)
```

Comment:
There are several places in this file using `url.parse` still. What not update those?

Reply:
Yeah, we should go through the entire codebase and update to the newer URL API. I wanted to address the performance issue in a timely manner with minimum risk. I'll log an issue to update to the new API.

Reply:
Gottchya---I assumed we'd be at risk with parsing these URLs differently if they return different results?

Reply:
`getPath` is only used in one location when resetting the [url](https://github.com/cypress-io/cypress/blob/02d04934725afd4a5b8067c1dd8ec1415e99ed3a/packages/server/lib/server-base.ts#L114). All the other functions in this file are used elsewhere and don't make direct comparisons to the url.

Reply:
logged https://github.com/cypress-io/cypress/issues/28369